### PR TITLE
Add $Fren token info

### PR DESCRIPTION
**Pull request by adakondacoin**

### CODE REVIEW

File: src/tokens/fren.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { defaultFetcherOptions, SupplyFetcher } from "../types";
+import { getAmountInAddresses, getBlockFrostInstance } from "../utils";
+
+const fren =
+  "fc11a9ef431f81b837736be5f53e4da29b9469c983d07f321262ce614652454e";
+
+const frenFetcher: SupplyFetcher = async (options = defaultFetcherOptions) => {
+  const blockFrost = getBlockFrostInstance(options);
+  const total = 420_069_000_000n;
```
Comment (TheTapHouse) [inline src/tokens/fren.ts:9]: change `const total = 420_069_000_000n;` -> `const total = 420_069_000_000;`

test fails due to different types (`TypeError: Cannot mix BigInt and other types, use explicit conversions`)

**Review by TheTapHouse — CHANGES_REQUESTED** [review]
Also import into `index.ts`

File: src/tokens/fren.ts (hunk 1 above)
Comment (adakondacoin) [reply src/tokens/fren.ts]: Thanks for the help.
What do I need to add in the index.ts in the dictionary<string, fetcher>?

???????????:frenFetcher

**Comment by adakondacoin** [comment]
Hey, what's wrong now?

**Comment by TheTapHouse** [comment]
`npm run format` updates index.ts to 
`  fc11a9ef431f81b837736be5f53e4da29b9469c983d07f321262ce614652454e: frenFetcher,`

without quotes around token unit

**Comment by adakondacoin** [comment]
Updated, thanks for the info

**Comment by adakondacoin** [comment]
Still not working

**Review by shadowkora — APPROVED** [review]

**Pull request merged**